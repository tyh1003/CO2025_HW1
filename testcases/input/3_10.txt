8
20 90 3 80 49 38 88 46
