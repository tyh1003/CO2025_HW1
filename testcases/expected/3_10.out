3 20 38 46 49 80 88 90 
